crates/core/tests/client.rs

```diff
@@ -30,6 +32,154 @@ mod tests {
         )
     }
 
+    #[tokio::test]
+    async fn test_starknet_event_to_eth_log_success() {
+        let client = setup_mock_client_crate().await;
+        // given
+        let data =
+            vec![0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 10];
+        let felt_data: Vec<FieldElement> =
+            data.iter().map(|&x| FieldElement::from_dec_str(&x.to_string()).unwrap()).collect();
+        let bytes_data: Bytes = felt_data.iter().flat_map(|felt| felt.to_bytes_be()).collect::<Vec<u8>>().into();
+        // see https://github.com/kkrt-labs/kakarot/blob/2133aaf58d5c8ae493c579570e43c9e011774309/tests/integration/solidity_contracts/PlainOpcodes/test_plain_opcodes.py#L120 this test generates the starknet event and ethereum log expected pair
+
+        // FROM is hardcoded to the current hardcoded value of kakarot_contract
+        let event3 = Event {
+            from_address: FieldElement::from_hex_be(
```

Comment:
I think that here, we would gain in making the test more verbose additionally to the above link, either:
- declare the variables ->
```rust
let kakarot_address = FieldElement::from_hex_be(0x566864dbc2ae76c2d12a8a5a334913d0806f85b7a4dccea87467c3ba3616e75).unwrap()
let event3 = Event {
from_address: kakarot_address,
// ...etc.
}
```

Or
- Adding comments above each hardcoded field to explain what it is.